cloudsensei/README.md

```diff
@@ -0,0 +1,71 @@
+## CloudSensei
+
+CloudSensei is a slack bot integrated with cloud governance, 
+allowing users to join a “read-only” slack channel to review 
+daily expense reports for instances (in running state) >= 7 days.
+We have had a few occurrences where cloud resources were 
+running indefinitely even after deleting the cluster. 
+But these few occurrences are good enough to be proactive and 
+this is where “CloudSensei” can give us quick access.
+
+### How it works?
+To implement this functionality, CloudSensei utilizes AWS Lambda, 
+a serverless computing service provided by Amazon Web Services. 
+AWS Lambda allows functions to be triggered without the need for managing 
+a dedicated CI (Continuous Integration) server. By leveraging Lambda, 
```

Comment:
I guess the intent here is to mention that with serverless you do not need to manage a dedicated server. Continuous Integration is not relevant in this context. perhaps remove that from this sentence?